MapboxNavigationUI/MGLMapView+MGLNavigationAdditions.h

```diff
@@ -0,0 +1,7 @@
+#import <Mapbox/Mapbox.h>
+
+@interface MGLMapView (MGLNavigationAdditions)
```

Comment:
Rather than redeclare the method below, let’s have this class extension conform to the CLLocationManagerDelegate protocol.